File: articles/automation-bias-and-human-overreliance/haskell/AutomationBiasRisk.hs
module Main where

main :: IO ()
main = do
  let acceptance = 0.93
      quality = 0.71
      uncertainty = 0.29
      reviewDeficit = 0.65
      overrideFriction = 0.72
      weakContestability = 0.0
      overrelianceGap = max 0 (acceptance - quality)
      score = (acceptance + overrelianceGap + uncertainty + reviewDeficit + overrideFriction + weakContestability) / 6
  putStrLn ("automation_bias_risk_score=" ++ show score)

File: articles/automation-bias-and-human-overreliance/rust/automation_bias_risk.rs
fn main() {
    let acceptance: f64 = 0.93;
    let quality: f64 = 0.71;
    let uncertainty: f64 = 0.29;
    let review_deficit: f64 = 0.65;
    let override_friction: f64 = 0.72;
    let weak_contestability: f64 = 0.0;
    let overreliance_gap = (acceptance - quality).max(0.0);
    let score = (acceptance + overreliance_gap + uncertainty + review_deficit + override_friction + weak_contestability) / 6.0;
    println!("automation_bias_risk_score={:.4}", score);
}
